packages/web-components/src/components/va-checkbox/va-checkbox.tsx

```diff
@@ -282,6 +282,14 @@ export class VaCheckbox {
                 {checkboxDescription}
               </span>
             )}
+            {hasInternalDescriptionSlot && (
+              <div
+                class="usa-checkbox__label-description"
+                part="internal-description"
+              >
```

Comment:
Do you mind clarifying a little more why the internal-description `slot` needs to be displayed conditionally with this extra div with class and part attributes? The slot won't render if there is no defined content within it.

Reply:
I had to move the `part="label"` to a span inside the label to prevent any CSS styles applied to it from modifying the entire label. When I added the internal slot, it needed that class to indent the content. I included the `part="internal-description"` in case that slot needed additional styling. I don't need this extra part, so I can remove it if it is unnecessary.